app/Services/Importers/AirportImporter.php

```diff
@@ -21,19 +21,34 @@ public function run($start = 0)
             'lat',
             'lng',
             'hub',
+            'ground_handling_cost',
+            'fuel_jeta_cost',
         ];
 
         $count = 0;
         $rows = $this->db->readRows($this->table, $this->idField, $start, $fields);
         foreach ($rows as $row) {
+            $ground_handling_cost = $row->ground_handling_cost;
+            $fuel_jetA_cost = $row->fuel_jeta_cost;
```

Comment:
You need to cast it here, otherwise your checks below (`!==`) won't work since they're strings

Reply:
Is it correct now?